src/mRides-server/Logic/UserCatalog.cs

```diff
@@ -144,18 +144,30 @@ public User create(User obj, int userId)
 
         public void setGSD(int userId, long amountGSD)
         {
-            //User u = _context.Users.Find(userId);
             User u = get(userId);
             u.GSD = amountGSD;
             _context.SaveChanges();
         }
 
         public long getGSD(int userId)
         {
-            //User u = _context.Users.Find(userId);
             User u = get(userId);
             return u.GSD;
         }
+
+        public string getGender(int userId)
+        {
+            User u = get(userId);
+            return u.gender;
+        }
+
+        public void setGender(int userId, string userGender)
+        {
```

Comment:
We should have an updateUser method that would take care of any attribute updates rather than a method for each attribute